content_scripts/desktopRenderer.js

```diff
@@ -100,7 +100,7 @@ import { UNIFORMS, MAX_VIEW_FRUSTUMS } from '../../src/gui/ViewFrustum.js';
 
             // try loading area target GLB file into the threejs scene
             isGlbLoaded = true;
-            let gltfPath =  realityEditor.network.getURL(object.ip, realityEditor.network.getPort(object), '/obj/' + object.name + '/target/target.glb');
+            let gltfPath =  realityEditor.network.getURL("localhost", realityEditor.network.getPort(object), '/obj/' + object.name + '/target/target.glb');
```

Comment:
Won't this cause problems if we connect remotely?

Reply:
Oh nevermind, this does look spicy but it's overwritten at the getURL level. I'll add a comment regarding that